cockroachcloud/frequently-asked-questions.md

```diff
@@ -84,22 +89,19 @@ See the [Security Overview page](../{{site.current_cloud_version}}/security-refe
 
 ### Is encryption-at-rest enabled on {{ site.data.products.dedicated }}?
 
-All data in {{ site.data.products.serverless }} and {{ site.data.products.dedicated }} is encrypted-at-rest by your chosen infrastructre-as-a-service provider, Google Cloud Platform (GCP) or Amazon Web Services (AWS), at the infrastructure level.
+All data in {{ site.data.products.serverless }} and {{ site.data.products.dedicated }} is encrypted-at-rest by your chosen infrastructre-as-a-service provider, Google Cloud Platform (GCP), Amazon Web Services (AWS), or Microsoft Azure (limited access), at the infrastructure level.
```

Comment:
```suggestion
All data in {{ site.data.products.serverless }} and {{ site.data.products.dedicated }} is encrypted-at-rest by your chosen infrastructure-as-a-service provider, Google Cloud Platform (GCP), Amazon Web Services (AWS), or Microsoft Azure (limited access), at the infrastructure level.
```

Reply:
I've updated this to "is encrypted at rest by each cloud platform at the infrastructure level." I don't think GCP is providing IaaS to the customer but to CRL based on the customer's choice.